chart/BUILD.yaml

```diff
@@ -14,9 +14,9 @@ packages:
       - imageRepoBase
     config:
       commands:
-        # set values
-        - ["yq", "w", "-i", "Chart.yaml", "version", "${version}"]
-        - ["yq", "w", "-i", "values.yaml", "version", "${version}"]
+        # set values. Use the value in values.yaml:.version as ${version}
+        - ["sh", "-c", "yq w -i Chart.yaml version $(yq r values.yaml version)"]
+        # - ["yq", "w", "-i", "values.yaml", "version", "${version}"]
```

Comment:
Disabling this line would break [branch builds](https://www.gitpod.io/docs/self-hosted/latest/install/install-on-kubernetes/#install-branch-build).

Installing branch builds builds works so nicely because the Helm charts that packaged into the installer has `version` properly set. Ad that's what the above LOC does.

Reply:
Thx, I was not aware we're using this already!

Then I propose we use `0.1.0-\<version\>`, as we do not have any reference to the "current version" in the repo left after https://github.com/gitpod-io/gitpod/pull/3192 got merged. Correct? @meysholdt